src/main/java/tanukidecor/block/TallBlock.java

```diff
@@ -157,11 +158,24 @@ public boolean canSurvive(BlockState pState, LevelReader pLevel, BlockPos pPos)
     }
 
     @Override
-    public void playerWillDestroy(Level pLevel, BlockPos pPos, BlockState pState, Player pPlayer) {
+    public BlockState playerWillDestroy(Level pLevel, BlockPos pPos, BlockState pState, Player pPlayer) {
         if (!pLevel.isClientSide && pPlayer.isCreative()) {
-            DoublePlantBlock.preventCreativeDropFromBottomPart(pLevel, pPos, pState, pPlayer);
```

Comment:
Again, prefer using methods that already exist in DoublePlantBlock instead of copy-pasting them.